parser/parser.go

```diff
@@ -533,20 +495,44 @@ func (p *parser) parseListValue() *List {
 	}
 }
 
-func (p *parser) parseMapValue() *Map {
-	lBracePos := p.scanner.Position
-	if !p.accept('{') {
-		return nil
+func (p *parser) parseMap(isModule bool) *Map {
```

Comment:
If we do want the parsing of a map within a module to permit different separator semantics than the parsing of a map outside a module, then it seems to me that the variable should be named based on what it does as opposed to who currently sets it (since the question of who sets it could change in the future).

So if we do keep the "isModule" parameter, I'd rename "isModule" to "requireCurlyBrace" (and then negate its value to match the name).

Hmm on closer inspection the name "isModule" is actually unrelated to your change so if you don't feel like worrying about that then nevermind